materialize-google-pubsub/transactor.go

```diff
@@ -34,7 +50,7 @@ func (t *transactor) Store(it *pm.StoreIterator) (pm.StartCommitFunc, error) {
 
 		msg := &pubsub.Message{
 			Data:        it.RawJSON,
-			OrderingKey: string(it.PackedKey), // Allows for reading of messages for the same key in order.
+			OrderingKey: fmt.Sprintf("%d", PackedKeyHash_HH64(it.PackedKey)),
```

Comment:
nity-nit: it's merged and not a big deal, but `%08x` would make these uniform size and lexicographically comparable: https://go.dev/play/p/D40eHGzzjWU